CoolNotifications/autorun/client/CoolNotifs.lua

```diff
@@ -0,0 +1,97 @@
+surface.CreateFont( "NoticeFont", {
+	font = "BITSTREAM VERA SANS", --Use the font-name which is shown to you by your operating system Font Viewer, not the file name
```

Comment:
make sure you use a font which is on gmod by default, other wise you have to add it manually
you can check them all out in `GarrysMod\garrysmod\resource\ClientScheme.res` under fonts